fiware_cloto/environments/environment_controller.py

```diff
@@ -35,23 +35,38 @@ class environment_controller():
 
     started = False
 
-    def check_python_process(self):
-        p = subprocess.Popen(['ps', '-awx'], stdout=subprocess.PIPE)
-        out, err = p.communicate()
 
-        for line in out.splitlines():
-            if 'environmentManager.py' in line:
-                self.started = True
-
-        return self.started
 
     def start_manager(self):
         if settings.SETTINGS_TYPE == 'production':
             arbiter = get_arbiter([{"cmd": "python "
                                        "" + settings.ENVIRONMENTS_MANAGER_PATH, "numprocesses": 1}], background=True)
 
-            if not self.check_python_process():
+            if not check_python_process():
+                arbiter.start()
```

Comment:
if you call arbiter.start() in both condition you should put it outside the if-else block, doesn't?